addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -294,13 +302,15 @@ export class Base {
     }
 }
 
-export function createRelatedModels(modelDefs, modelClasses = {}, indexes = {}, database = {}) {
+export function createRelatedModels(modelDefs, modelClasses = {}, opts = {}) {
+    const indexes = opts.databaseIndex || {};
+    const database = opts.databaseTable || {};
     const [inverseMap, processedModelDefs] = processModelDefs(modelDefs);
-    const records = mapObj(processedModelDefs, () => reactive({}));
-    const orderedRecords = mapObj(processedModelDefs, () => reactive([]));
+    const records = mapObj(processedModelDefs, () => reactive(new Map()));
     const callbacks = mapObj(processedModelDefs, () => []);
     const commands = mapObj(processedModelDefs, () => ({
         delete: new Map(),
+        unlink: new Map(),
```

Comment:
what's the difference between unlink and delete?

Reply:
Unlink will just unlink the record in the backend and delete will delete it